factory/usr/lib/systemd/system/initrd-switch-root.service.d/core-override.conf

```diff
@@ -1,18 +0,0 @@
-[Service]
```

Comment:
can we implement @xnox's suggestion and not delete this file in this PR, just add the rules file, and then in the next PR where you re-work the unit dependencies (https://github.com/snapcore/core-initrd/pull/59) delete this file? AFAICT adding the udev file without deleting this file won't break anything

Reply:
I have changed this PR. Will put the remaining change in #59